app/assets/stylesheets/donor.css.scss

```diff
@@ -0,0 +1,229 @@
+@mixin clear-list-margins() {
+  margin: 0;
+  list-style-type: none;
+}
+
+
+.profile-summary-donate-link {
+  background: $white;
+  border: 1px solid $donor-profile-border-grey;
+  border-radius: 4px;
+  color: $medium-green;
+  margin: 0 auto;
+  padding: 5px 10px;
+}
+
+.update-profile-link {
+  background: $donor-profile-light-grey;
+  color: $donor-profile-darker-grey;
+  padding: 8px;
+  text-decoration: none;
+  width: 100px;
+}
+
+.donor-dashboard-block {
+  @include full-width-block($off-white);
+  @include block-padding();
+  min-height: 900px;
+
+  .dashboard {
+    border: 1px solid $dark-grey;
+    @include drop-shadow(1);
+
+    .card-header {
+      @extend .text-center;
+      border-bottom: 1px solid $dark-grey;
+      background-color: $medium-green;
+
+      .card-header-title {
+        display: inline-block;
+        font-size: rem-calc(18);
+        font-weight: 400;
+        color: $white;
+        margin-top: 8px;
+        margin-bottom: 8px;
+      }
+
+      @mixin card-header-arrow() {
+        display: inline-block;
+
+        i.fa {
+          padding-top: 12px;
+        }
+
+        a {
+          color: $dark-grey;
+          &:hover {
+            color: $white;
+          }
+        }
+      }
+
+      .card-header-arrow-left {
+        @include card-header-arrow();
+        margin-right: 30px;
+      }
+
+      .card-header-arrow-right {
+        @include card-header-arrow();
+        margin-left: 30px;
+      }
+    }
+
+    .card-profile {
+      padding-top: 30px;
+      background-color: $white;
+      border-bottom: 1px solid $dark-grey;
+      color: $charcoal;
+
+      .profile-info {
+        .profile-img-wrap {
+          @include clear-list-margins();
+
+          .profile-img-item {
+            text-align: center;
+            .profile-img {
+              max-height: rem-calc(200px);
+              padding: rem-calc(15px);
+            }
+          }
+        }
+
+        .profile-summary-wrap {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5